docs/serving/samples/hello-world/helloworld-go/Dockerfile

```diff
@@ -1,24 +1,25 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.13 as builder
```

Comment:
Any particular reason not to use golang:1?

We have to keep updating this every 6 months otherwise.

Reply:
Given that sometimes the minor version has significant change to best practices (e.g., modules) I think the clarity is worthwhile for the reader and motivation to update.